extra_file.py

```diff
@@ -0,0 +1,74 @@
+import re
+import datetime
+from collections import OrderedDict
+
+def create_racer_abbreviations_dict(file_name):
+    """Retrieves {'abbreviation': (name, team)}" format dict from abbreviations.txt"""
```

Comment:
Please, follow docstring style guides.
As an example take a look ad Google style guide for docstrings here https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html